src/Scoper.php

```diff
@@ -50,6 +51,7 @@ public function scope(string $content, string $prefix): string
     {
         $traverser = new NodeTraverser();
         $traverser->addVisitor(new ParentNodeVisitor());
+        $traverser->addVisitor(new SingleLevelUseAliasVisitor($prefix));
```

Comment:
I don't think it matters since the enterNode() needs the check a UseUse Name child node - the order is bound to parent first, child second no matter the order of the visitors.